e3sm_diags/driver/utils/regrid.py

```diff
@@ -406,6 +374,8 @@ def align_grids_to_lower_res(
 
     is_a_lower_res = len(axis_a) <= len(axis_b)
 
+    # TODO: Potentially not needed if we just address the bad bound with the
+    # Hadley SST dataset.
```

Comment:
Pending @chengzhuzhang

Reply:
@tomvothecoder I have updated the Hadley SST data at /lcrc/group/e3sm/diagnostics.